src/dsp_tools/utils/connection_live.py

```diff
@@ -335,49 +263,46 @@ def _log_and_raise_timeouts(self, error: TimeoutError | ReadTimeout) -> Never:
     def _log_response(self, response: Response) -> None:
         dumpobj: dict[str, Any] = {
             "status_code": response.status_code,
-            "headers": self._anonymize(dict(response.headers)),
+            "headers": _sanatize_headers(dict(response.headers)),
         }
         try:
-            dumpobj["content"] = self._anonymize(json.loads(response.text))
+            dumpobj["content"] = response.json()
         except json.JSONDecodeError:
-            dumpobj["content"] = response.text if "token" not in response.text else "***"
+            dumpobj["content"] = response.text
         logger.debug(f"RESPONSE: {json.dumps(dumpobj)}")
 
-    def _anonymize(self, data: dict[str, Any] | None) -> dict[str, Any] | None:
-        if not data:
-            return data
-        data = data.copy()
-        if "token" in data:
-            data["token"] = self._mask(data["token"])
-        if "Set-Cookie" in data:
-            data["Set-Cookie"] = self._mask(data["Set-Cookie"])
-        if "Authorization" in data:
-            if match := regex.search(r"^Bearer (.+)", data["Authorization"]):
-                data["Authorization"] = f"Bearer {self._mask(match.group(1))}"
-        if "password" in data:
-            data["password"] = "*" * len(data["password"])
-        return data
-
-    def _mask(self, sensitive_info: str) -> str:
-        unmasked_until = 5
-        if len(sensitive_info) <= unmasked_until * 2:
-            return "*" * len(sensitive_info)
-        else:
-            return f"{sensitive_info[:unmasked_until]}[+{len(sensitive_info) - unmasked_until}]"
-
-    def _in_testing_environment(self) -> bool:
-        in_testing_env = os.getenv("DSP_TOOLS_TESTING")  # set in .github/workflows/tests-on-push.yml
-        return in_testing_env == "true"
-
     def _log_request(self, params: RequestParameters) -> None:
         dumpobj = {
             "method": params.method,
             "url": params.url,
-            "headers": self._anonymize(dict(self.session.headers) | (params.headers or {})),
+            "headers": _sanatize_headers(dict(self.session.headers) | (params.headers or {})),  # type: ignore[operator]
             "timeout": params.timeout,
         }
         if params.data:
-            dumpobj["data"] = self._anonymize(params.data)
+            data = params.data.copy()
+            if "password" in data:
+                data["password"] = "***"
+            dumpobj["data"] = data
         if params.files:
             dumpobj["files"] = params.files["file"][0]
         logger.debug(f"REQUEST: {json.dumps(dumpobj, cls=SetEncoder)}")
+
+
+def _sanatize_headers(headers: dict[str, str | bytes]) -> dict[str, str]:
+    def _mask(k: str, s: str | bytes) -> str:
```

Comment:
would be more readable if you used proper names not just one letter

Reply:
I tend to not mind short variables when they have such a small scope. But you're right, it's more readable otherwise: https://github.com/dasch-swiss/dsp-tools/pull/1238/commits/891bce8935808997e10cac4263e932e87ac5adb9